portal/contracts/vote.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Vote contract
+author: "Raza Zaidi"
+authorAvatar: "/assets/portal/authors/raza.jpeg"
+authorTwitter: "https://twitter.com/razacodes"
+draft: false
+summary: ""
+date: "2021-03-29"
+updated: ""
+---
+
+## In a nutshell
+
+The **Vote** contract is a governance contract. It's a smart contract that faciliates the decision-making process in a community. 
+The most common usecase is within a DAO, where proposals are made and DAO-members cast their **vote**. DAO means Decentralised Autonomous Organization.
+It's an entity that's made up of several smart contracts (on-chain). A DAO is often used by a dedicated community that needs an infrastructure to operate the organization.
+A DAO has several components, but the most prominent component is the treasury, which is managed and used by the whole community.
+
+## How it works
+
+In order to allow members to vote, they need to hold a so-called `governance token`. A `governance token` is an ERC-20 token. 
+The **Vote** contract needs an ERC-20 contract to function. With thirdweb you can also deploy your very own ERC-20 contract, the **Token** contract.
+You can find more about the **Token** contract [here](/contracts/token)
+
+
+## What can you do with our contract?
+
+The following use cases are suited for the Vote contract:
+
+1. You are working with a team and need to decide on organizational changes
+2. Voting on managing and distributing funds in a treasury
+3. Vote on any other proposal in an organization, such as a DAO.
+
+## Setting up your Vote contract
+
+You can create a **Vote** contract from the thirdweb dashboard. Go ahead and click on `Create new contract`.
+![Create new contract](/assets/portal/contracts/vote/1_create_contract.png)
+
+The **Vote** contract can be found under Governance.
+![Create new contract](/assets/portal/contracts/vote/2_select_governance.png)
+![Create new contract](/assets/portal/contracts/vote/3_select_vote.png)
+
+A **Vote** contract allows you to configure the following settings:
+1. `Governance Token Address` is an ERC-20 contract (our **Token** contract) to work with the **Vote** contract.
```

Comment:
it can be our Token contract or any contract that implements IVotes interface: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.5.0/contracts/governance/utils/IVotes.sol

Also, when mentioning other contracts, link to them. In this case [Token](/contracts/token)

Reply:
adjusted